core/miner/storage_fsm/sealing_events.hpp

```diff
@@ -164,6 +166,15 @@ namespace fc::mining {
     types::SectorPreCommitInfo precommit_info;
   };
 
+  struct SectorPreCommittedBatchContext final : public SealingEventContext {
+   public:
```

Comment:
not necessary for `struct`

Reply:
it is not necessary, but will be strict and obvious in given approach

Reply:
It is already obvious by `struct` keyword